dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -98,11 +100,38 @@ public static void main(String[] arguments)
       throws IOException, TemplateException, RepositoryException, URISyntaxException,
           ParseException, MavenRepositoryException {
     DashboardArguments dashboardArguments = DashboardArguments.readCommandLine(arguments);
-    Path output =
-        dashboardArguments.hasFile()
-            ? generate(dashboardArguments.getBomFile())
-            : generate(dashboardArguments.getBomCoordinates());
-    System.out.println("Wrote dashboard into " + output.toAbsolutePath());
+
+    if (dashboardArguments.hasVersionlessCoordinates()) {
+      generateAllVersions(dashboardArguments.getVersionlessCoordinates());
+    } else {
+      Path output =
+          dashboardArguments.hasFile()
+              ? generate(dashboardArguments.getBomFile())
+              : generate(dashboardArguments.getBomCoordinates());
+      System.out.println("Wrote dashboard into " + output.toAbsolutePath());
```

Comment:
Why not have `generate()` print this? The one with the file could write "Wrote dashboard for _file_ into _output_", and the one with coordinates could write "Wrote dashboard for _coordinates_ into _output_". Then `generateAllVersions` doesn't have to do it too.

Reply:
Nice idea. Updated.